pause-on-lock

```diff
@@ -33,6 +33,10 @@ elif [ "$XDG_CURRENT_DESKTOP" == "X-Cinnamon" ] ; then
   DBUS_LOCK="interface='org.cinnamon.ScreenSaver',member='ActiveChanged'"
   LOCK_SIGNAL="true"
   UNLOCK_SIGNAL="false"
+elif [ "$XDG_CURRENT_DESKTOP" == "MATE" ] ; then
+  DBUS_LOCK="interface='org.mate.ScreenSaver',member='ActiveChanged'"
+  LOCK_SIGNAL="true"
+  UNLOCK_SIGNAL="false"
 else
 echo "Error: Unsupported desktop. Supported desktops are Unity, GNOME and Cinnamon."
```

Comment:
One last line to change, after that everything should be fine.
```suggestion
echo "Error: Unsupported desktop. Supported desktops are Unity, GNOME, Cinnamon and MATE."
```

Reply:
Done!